### ISSUE TITLE
java.lang.IllegalArgumentException: Cannot add a null child view to a ViewGroup

### ISSUE THREAD

**Issue by isamotiuc** [AdGem/Android-Example#4]
Hello, 
After update to 0.9.8 one of the issue was increased. As we see in crashlytics is old one, but it starts to increase.

```
Fatal Exception: java.lang.RuntimeException: Unable to start activity ComponentInfo{co.giftloop/com.adgem.android.internal.offerwall.OfferWallActivity}: java.lang.IllegalArgumentException: Cannot add a null child view to a ViewGroup
       at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2660)
       at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2721)
       at android.app.ActivityThread.-wrap12(ActivityThread.java)
       at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1469)
       at android.os.Handler.dispatchMessage(Handler.java:102)
       at android.os.Looper.loop(Looper.java:154)
       at android.app.ActivityThread.main(ActivityThread.java:6142)
       at java.lang.reflect.Method.invoke(Method.java)
       at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:867)
       at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:757)
```
![image](https://user-images.githubusercontent.com/2742322/49084352-e9991380-f257-11e8-850a-6d83d8dcada6.png)

Full stack trace can be found [here](https://gist.github.com/isamotiuc/8fe95f4c2b64666eecfe39562ba6bcf0)

Thank you.

**Comment by Exaper** [comment]
@isamotiuc thanks for this report. Could you please upgrade to AdGem v1.0.5 and see if issue still exists? Thanks!

**Comment by isamotiuc** [comment]
@isamotiuc Will keep you in touch after update 👍

**Comment by Exaper** [comment]
@isamotiuc Happy New Year! We've shipped SDK v1.1.2 which should have OfferWall fixed for you. Can you take a look at updated version? Thanks!

**Comment by isamotiuc** [comment]
Thank you 👍  will update this issue after new our release
